casts/countryPicker/src/UICountryPicker/CountryPicker.tsx

```diff
@@ -121,6 +129,13 @@ export function CountryPicker({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    const hideKeyboard = React.useCallback(() => {
+        // react-native-android-keyboard-adjust bug:
+        // Keyboard doesn't want to hide on Android
+        // so we have to forcibly hide the keyboard
+        isAndroid && focused && Keyboard.dismiss();
```

Comment:
Btw AFAIK you could safely call `Keyboard.dismiss()` without checking whether it's focused or not